releases/tickets-source/v15_0.rst

```diff
@@ -1,393 +1,271 @@
+- `DM-11735 <https://jira.lsstcorp.org/browse/DM-11735>`_: fix tearDown race conditions in obs_test [obs_test]
+- `DM-11758 <https://jira.lsstcorp.org/browse/DM-11758>`_: Make  WarpType configurable in GetCoaddAsTemplateTask [ip_diffim]
+- `DM-11764 <https://jira.lsstcorp.org/browse/DM-11764>`_: Fix pickling support in daf_base for pybind11 2.2.0 [daf_base]
+- `DM-11771 <https://jira.lsstcorp.org/browse/DM-11771>`_: obs_base fails to build on NFS-mounted systems [daf_persistence, obs_base]
+- `DM-11776 <https://jira.lsstcorp.org/browse/DM-11776>`_: "RuntimeWarning: invalid value encountered in greater/less" in processing HSC data [afw, ip_isr, meas_algorithms, pipe_tasks]
+- `DM-11781 <https://jira.lsstcorp.org/browse/DM-11781>`_: flake8 line length checking is catching shebangs [sconsUtils]
+- `DM-11790 <https://jira.lsstcorp.org/browse/DM-11790>`_: Move jointcal doc to DMTN-36 [jointcal]
+- `DM-11799 <https://jira.lsstcorp.org/browse/DM-11799>`_: Confusing log messages about exitStatus in successful makeSkyMap tasks [pipe_tasks]
+- `DM-11809 <https://jira.lsstcorp.org/browse/DM-11809>`_: Explicitly enable pep8-naming tests [base, meas_base, sconsUtils, shapelet, utils, verify]
+- `DM-11810 <https://jira.lsstcorp.org/browse/DM-11810>`_: meas_algorithms fails on NFS-mounted systems [meas_algorithms]
+- `DM-11853 <https://jira.lsstcorp.org/browse/DM-11853>`_: shebang target should run before tests [sconsUtils]
+- `DM-11861 <https://jira.lsstcorp.org/browse/DM-11861>`_: SqlRegistry raising exception due to missing super.__del__ [daf_persistence]
+- `DM-11862 <https://jira.lsstcorp.org/browse/DM-11862>`_: writeFitsCatalogStorage calls obj.writeFits incorrectly [daf_persistence]
+- `DM-11871 <https://jira.lsstcorp.org/browse/DM-11871>`_: afw.table column access is slow via__getitem__ (as compared to get) [afw]
+- `DM-11885 <https://jira.lsstcorp.org/browse/DM-11885>`_: look in daf_persistence for python2-style calls to super() and replace with python 3 style calls. [daf_persistence]
+- `DM-11894 <https://jira.lsstcorp.org/browse/DM-11894>`_: Speed up SpanSet operations [afw]
+- `DM-11895 <https://jira.lsstcorp.org/browse/DM-11895>`_: Support getting data for current card of FitsChan [astshim]
+- `DM-11905 <https://jira.lsstcorp.org/browse/DM-11905>`_: Fix PhotoCalib defintion to use multiplication [afw]
+- `DM-11911 <https://jira.lsstcorp.org/browse/DM-11911>`_: Update obs_ctio0m9 to work following DM-5922 [obs_base]
+- `DM-11916 <https://jira.lsstcorp.org/browse/DM-11916>`_: Warnings "Extent2I object has no attribute getWidth" when running meas_mosaic, jointcal [jointcal]
+- `DM-11917 <https://jira.lsstcorp.org/browse/DM-11917>`_: Error in coaddDriver following compression changes [pipe_drivers]
+- `DM-11927 <https://jira.lsstcorp.org/browse/DM-11927>`_: Set maxOffsetPix consistently [obs_subaru]
+- `DM-11929 <https://jira.lsstcorp.org/browse/DM-11929>`_: add `from builtins import super` to registries.py [daf_persistence]
```

Comment:
We probably don't want the backticks here (or we want to convert them to double-backticks): this is being interpreted as a restructured text role.